gap/Tools.gd

```diff
@@ -112,7 +112,7 @@ DeclareOperation( "CoimageObjectFunctorialWithGivenCoimageObjects",
 
 ###################################
 ##
-#! @Section PreInverse and PostInverse
+#! @Section Projective Covers and Injective Envelops
```

Comment:
I'm not sure if AutoDoc is case-sensitive, but the casing differs from the one in Doc.autodoc.